aikau/src/main/resources/alfresco/util/domUtils.js

```diff
@@ -121,6 +129,67 @@ define([
 
             // Pass back the listener-removal object
             return listenerRemovalObj;
+         },
+
+         // See API below
+         addPollingResizeListener: function alfresco_util_domUtils__addPollingResizeListener(nodeToMonitor, resizeHandler) {
+
+            // Add the new monitoring nodes
+            var nodes = resizeMonitor.nodes,
+               monitorKey = Date.now();
+            while (nodes.hasOwnProperty(monitorKey)) {
+               monitorKey = Date.now();
+            }
+            nodes[monitorKey] = {
+               domNode: nodeToMonitor,
+               height: nodeToMonitor.offsetHeight,
+               width: nodeToMonitor.offsetWidth,
+               callbackFunc: resizeHandler
+            };
+
+            // If there is no monitoring running, start it up
+            if (!resizeMonitor.removeObj) {
+               resizeMonitor.removeObj = funcUtils.addRepeatingFunction(this._checkNodesForResize, "SHORT");
```

Comment:
Oh, and it didn't seem important enough to have its own enums class ... unless you think it could??